appstudio-utils/util-scripts/lib/fetch/cluster.sh

```diff
@@ -55,6 +55,9 @@ save-policy-config() {
     local namespace=${namespace_arg#-n }
     echo "ERROR: unable to find the ec-policy EnterpriseContractPolicy in namespace ${namespace:-$(kubectl config view --minify -o jsonpath='{..namespace}')}" 1>&2
   else
+    if [[ -z "${non_blocking_data}" ]]; then
```

Comment:
I'm very disappointed jq doesn't have a way of handling this. I tried the alternative operator, `//`, but jq seems to just do nothing if the input is empty.

Reply:
Ah! A bit wonky but works:

```
🐚 echo '["foo"]' | jq -s '.[0] // [] | {"config": {"policy": {"non_blocking_checks": . }}}'
echo '["foo"]' | jq -s '.[0] // [] | {"config": {"policy": {"non_blocking_checks": . }}}'
{
  "config": {
    "policy": {
      "non_blocking_checks": [
        "foo"
      ]
    }
  }
}

🐚 echo | jq -s '.[0] // [] | {"config": {"policy": {"non_blocking_checks": . }}}'
{
  "config": {
    "policy": {
      "non_blocking_checks": []
    }
  }
}

```

What you have is fine and probably more readable. I just couldn't let this go :grin: